PennMobile/Setup + Navigation/RootViewController.swift

```diff
@@ -133,6 +128,25 @@ class RootViewController: UIViewController, NotificationRequestable {
             
         }
         
+        //Request to fetch Two Factor Code again if the app failed to fetch it last time
+        if shouldFetchTwoFactorCode() {
```

Comment:
We need to re-architect how we display pop-ups. Right now, we have a series of if statements, each individually checking if it should display its popup. In the short-term, we should maybe make them if-else statements so that no more than one gets run on a single app open. My concern is that the two-factor code popup will show up on top of the courses privacy screen.